crates/node-bindings/Cargo.toml

```diff
@@ -7,12 +7,20 @@ edition = "2021"
 [lib]
 crate-type = ["cdylib"]
 
+[features]
+canary = []
+
 [dependencies]
 napi-derive = "2.12.5"
 parcel-js-swc-core = { path = "../../packages/transformers/js/core" }
 parcel-resolver = { path = "../../packages/utils/node-resolver-rs" }
 dashmap = "5.4.0"
 xxhash-rust = { version = "0.8.2", features = ["xxh3"] }
+sentry = "0.32.2"
```

Comment:
nit:

if you only use these dependencies for sentry you can make them optional then add them into the `canary = []` array to only compile them when this feature is enabled.

you'd first make `once_cell = { version = "1.19.0", optional = true }` then add to the array; see "optional dependencies" https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies